### ISSUE TITLE
Allow short and partial issue/PR URIs

### ISSUE THREAD

**Issue by jayvdb** [coala/corobo#350]
i.e. assign & mark should accept `foo/bar#1`

**Comment by meetmangukiya** [comment]
Would be done via https://github.com/coala/corobo/issues/167

**Comment by jayvdb** [comment]
No, this isnt about `file issue` ; this is assign and unmark, and `foo/bar#1` is precise.  #167 is about guessing the correct repo, and that is what the patch being developed is doing.

**Comment by yashasingh** [comment]
@jayvdb I would like to take up this issue.

**Comment by Makman2** [comment]
Code can (and should) be reused from my PR at https://github.com/coala/corobo/pull/297
However testing is the problem, as the test frontend doesn't support needed fields (which seem to exist in the gitter frontend).
